api/core/v2/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/sensu/sensu-go/api/core/v2
+
+go 1.14
```

Comment:
I thought there were issues preventing us from upgrading to go 1.14?

Reply:
That's true, I should adjust this to be go 1.13.